articles/introduction-to-elastic-stack/index.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: engineering-education
+status: publish
+published: 
+url: /engineering-education/introduction-to-elastic-stack/
+title: Introduction to Elastic Stack(ELK)
+description: This article will cover the basics of Elastic Stack - a log management platform
+author: elly-omondi
+date: 2021-11-08T00:00:00-01:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+
+   - url: /engineering-education/introduction-to-elastic-stack/hero.jpg
+    alt: ELK cover image 
+---
+
+
+[**Elastic stack** (ELK)](https://www.elastic.co/elastic-stack)  is a bundle of software components that can search, visualize, and manage log files(end to end log analysis).
+<!--more-->
+
+Computing devices perform activities and processes that upon completion result into logs that are stored as files. These log data can be  present in servers(i.e. from completed user authentication processes) as well as mobile devices (i.e from activity log)
+The Elastic stack(ELK) provides a log management platform of these log files
+
+
+
+### Overview
+This article will cover:
+- [What is Elastic Stack](#What-is-elastic-stack)
+- [Components of the Elastic Stack](#Components-of-the-elastic-stack)
+- [How ELK Works](#CHow-ELK-works)
+- [Benefits of using ELK](#Benefits-of-using-ELK)
+
+### What is Elastic Stack
+
+Elastic stack, popularly known as the ELK, is a bundle of software components(tools) that provide the ability to search, view and manage log files. The ELK does the processing of the log files (data in any format) from all systems and applications in use then stores them in a JSON formatted document.
+The ELK gives users the ability to analyze these  logs to create meaningful visualisations, perform infrastructure and system monitoring, do efficient troubleshooting and even identify and raise security issues.
+
+
+The software  suite is widely used because it is open-sourced (both for personal and entreprise use) and it can be scaled easily based on users needs
+There exist also paid version of ELK that are hosted in the cloud.
+
+
+### 
+For many years, the Elastic Stack existed as a collection of 3 tools, hence the acronym ELK. Recently, a fourth component was added to the suite.
```

Comment:
You can add the components of ELK in this sentence where you mention it 'a collection of 3 tools'. And, maybe move the bullets right after the sentence.